CONTRIBUTING.md

```diff
@@ -122,6 +122,13 @@ The folder structure boils down to:
 
 All new code should follow the folder structure.
 
+### How to add a new example
+
+1. Add the example in the ["List of all examples"](./doc/website/getting-started/list-of-examples.md)
+2. Create a new file and include the readme in `doc/website/getting-started/examples/ice-foo.md`
```

Comment:
I think this is not very clear. Basically you want a new readme to be added to this folder `doc/website/getting-started/examples/`, right?! It's also not clear if it always has to start with `ice`. Also, should the readme go here or will the file included here basically just point to a readme in the folder of the example?

Reply:
Jep, examples are included to have a single source. Clarified the text